website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Concepts"
+linkTitle: "Concepts"
+weight: 35
+---
+
+By adding Karpenter to a Kubernetes cluster you can dramatically improve the efficiency and cost of running workloads on that cluster.
+Instead of introducing lots of new concepts and configuration, Karpenter is tightly integrated with Kubernetes features to make sure that the right types and amounts of compute resources are available to pods as they are needed.
+Karpenter carries out its duties by:
+
+* Watching for pods that the Kubernetes scheduler has marked as unschedulable
+* Evaluating the requirements (capacity, features, and taints) identified by the unscheduable pods
+* Provisioning nodes that meet the requirements of the unschedulable pods
```

Comment:
We repeat "unschedulable" here a bunch, maybe we can just say it once in the first bullet and then the context can be inferred and simplified to"the pods"